examples/pxScene2d/src/pxScene2d.cpp

```diff
@@ -2288,10 +2344,13 @@ rtDefineObject(pxScene2d, rtObject);
 rtDefineProperty(pxScene2d, root);
 rtDefineProperty(pxScene2d, w);
 rtDefineProperty(pxScene2d, h);
+rtDefineProperty(pxScene2d, showpxObjCount);
+rtDefineProperty(pxScene2d, showTexMemUsage);
 rtDefineProperty(pxScene2d, showOutlines);
 rtDefineProperty(pxScene2d, showDirtyRect);
 rtDefineMethod(pxScene2d, create);
 rtDefineMethod(pxScene2d, clock);
+rtDefineMethod(pxScene2d, gc);
```

Comment:
I think it's best to only add one method added called logDebugMetrics() as opposed to exposing the object count, texture memory usage, and garbage collection.  Internally, this method will trigger a garbage collection, log the pxObject count, and log the texture memory usage.